challenges.yml

```diff
@@ -0,0 +1,184 @@
+addsimplenamespacecoding:
```

Comment:
Ah, this is perfect actually! These keys should be based off of the original class names:

* `Challenges\AddSimpleNamespaceCoding` -> `addsimplenamespacecoding` (exactly as you have)
* (if there is a namespace) `Challenges\Foo\SomeChallenge` -> `foo_somechallenge`

If we do this, then if a user previously answered a challenge, KnpU.com will remember that they got it right (it links the old challenge to the new one).